src/ensembl/src/content/app/browser/track-panel/track-panel-list/TrackPanelList.tsx

```diff
@@ -103,14 +108,31 @@ const TrackPanelList: FunctionComponent<TrackPanelListProps> = (
     };
   };
 
-  const getTrackListItem = (track: TrackPanelItem | null) => {
+  const getDefaultTrackStatus = (categoryName: string, trackName: string) => {
+    const statesOfCategory = props.trackStates[categoryName];
+
+    let defaultTrackStatus = ImageButtonStatus.ACTIVE;
```

Comment:
Very confused why a track status is defined by an image button status.

Reply:
That's what was used previously and I thought of keeping it for now. But I understand what you mean. This is semantically wrong and I probably need to create a new enum for the track states.